docs/source/models/concept_models/vivarium_mncnh_portfolio/antenatal_care_module/module_document.rst

```diff
@@ -137,64 +181,42 @@ The below table describes what propensity values to use for each exposure option
-    - GBD covariate ID 7: :code:`get_covariate_estimates(location_id=location_id, release_id=16, year_id=2023, covariate_id=7)` 
-    - Used to calculate propensity values for ANC coverage
-    - 
-  * - ANC4
-    - GBD covariate ID 8: :code:`get_covariate_estimates(location_id=location_id, release_id=16, year_id=2023, covariate_id=8)` 
-    - Used to calculate propensity values for ANC coverage
-    - 
-
 2.3 Module Outputs
-++++++++++++++++++
+------------------
 
 As mentioned earlier, ANC attendance impacts hemoglobin exposure and facility choice in our model, and in order for the 
 outputs of this component to be compatible with the data needs of these two downstream components, we will need two different
-outputs, one being dichotomous for the hemoglobin component and the other being polychotomous for the facility choice component. 
-
-ANC attendance inputs to the :ref:`hemoglobin module <2024_vivarium_mncnh_portfolio_hemoglobin_module>`
-are dichotomous for each pregnancy, so we need to observe the following: 
+outputs, one being dichotomous for the hemoglobin component and the other being polychotomous for the facility choice component.  
 
 .. list-table:: Module outputs for hemoglobin component
   :header-rows: 1
+  :widths: 10 15 15
 
   * - Output
     - Value
     - Note
   * - Attends ANC in first trimester?
-    - *True* / *False*
-    - For groups B or D 
+    - 
+      - *True*  for groups A and B 
+      - *False* for groups C and D
+    - This output will be used as an input for the :ref:`hemoglobin module <2024_vivarium_mncnh_portfolio_hemoglobin_module>`.
+      This variable is dichotomous for each pregnancy.
   * - Attends ANC in later pregnancy?
-    - *True* / *False* 
-    - For groups A or C
-
-ANC attendance inputs to the :ref:`facility choice module <2024_vivarium_mncnh_portfolio_facility_choice_module>`
-are polychotomous for each pregnancy with the following 4 categories (ordered from worst to best):
-
-1. No ANC
-2. ANC in later pregnancy only
-3. ANC in 1st trimester only
-4. ANC in 1st trimester and later pregnancy
+    - 
+      - *True*  for groups A and C 
+      - *False* for groups B and D
+    - This output will be used as an input for the :ref:`hemoglobin module <2024_vivarium_mncnh_portfolio_hemoglobin_module>`.
+      This variable is dichotomous for each pregnancy.
+  * - ANC attendance 
+    - 
+      1. No ANC
+      2. ANC in later pregnancy only
+      3. ANC in 1st trimester only
+      4. ANC in 1st trimester and later pregnancy
+    - This output will be used as an input for the :ref:`facility choice module <2024_vivarium_mncnh_portfolio_facility_choice_module>`.
+      The variable is polytomous for each pregnancy with the above 4 categories (ordered from worst to best). These are the same four categories listed above as A-D, 
+      but in reverse order, i.e., 1 = D, 2 = C, 3 = B, 4 = A. These need to be ordered D < C < B < A when sampling the variable using the correlated propensity for 
```

Comment:
I like the new table format!

I have a few comments:

- "the above 4 categories" no longer quite applies since they're listed in the table instead of "above"
- I think it's worth specifying exactly what we want the categories to be called. My proposal would be (to avoid the redundancy of saying "ANC" in all the category names):

1. `none` for group D
2. `later_pregnancy_only` for group C
3. `first_trimester_only` for group B
4. `first_trimester_and_later_pregnancy` for group A

- I think we can simplify the explanation in the Note column, for example, by explicitly specifying which category corresponds to which letter in the value column, like I've done above. Perhaps I'll just try making a code suggestion...